Things_You_Should_Know/kind_property.md

```diff
@@ -0,0 +1,29 @@
+# App Service Kind property
+
+Everything in Azure is modeled as a **resource**. You can learn more about resources through [Azure Resource Manager documentation](https://docs.microsoft.com/azure/azure-resource-manager/management/overview)
+
+Every Azure resource has a set of generic properties that are common across all resources. **kind** is one of this generic azure resource properties. See [Resources - Get](https://docs.microsoft.com/rest/api/resources/resources/get)
+
+There are several different offerings in the azure portal that use the same underlying `Microsoft.web\sites` resource. [Learn more about `microsoft.web\sites`](https://docs.microsoft.com/azure/templates/microsoft.web/2019-08-01/sites)
```

Comment:
In general, should this be `Microsoft.Web\Sites` ?

Reply:
Fixed